.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**Description:**
```

Comment:
Small suggestion to improve the issue template including categories etc. similar to base products

Reply:
I checked the issue management in base products and we have some work to do ( deciding on issue types, template for each type etc) if we are to go in that path. Therefore shall we do that from an upcoming effort?